src/applications/static-pages/simple-forms/21-4138/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can submit this form online or by mail.</p>
+        <a
+          className="vads-c-action-link--blue"
+          href="/supporting-forms-for-claims/support-statement-21-4138"
+        >
+          Submit a request online for priority processing for an existing claim
```

Comment:
Is this the right verbiage for this part? I thought Priority Processing was a different form.

Reply:
Oops, good catch. Fixed!